server/src/main/java/com/google/musicanalysis/types/AnalysisPair.java

```diff
@@ -4,9 +4,9 @@
 
 public class AnalysisPair {
   public final HashMap<String, String> perspectiveMap;
-  public final MagnitudeAndScore magnitudeAndScore;
+  public final NLPResult magnitudeAndScore;
 
-  public AnalysisPair(HashMap<String, String> perspectiveMap, MagnitudeAndScore magnitudeAndScore) {
+  public AnalysisPair(HashMap<String, String> perspectiveMap, NLPResult magnitudeAndScore) {
     this.perspectiveMap = perspectiveMap;
     this.magnitudeAndScore = magnitudeAndScore;
   }
```

Comment:
run `npm run lint:fix` in `client` to autofix lint errors?

Reply:
coming in front-end pr...since this one just has backend code it will put in a bunch of new files